rest-service/manager_rest/constants.py

```diff
@@ -89,3 +88,34 @@
               'csys-obj-parent'}
 
 CFY_LABELS_PREFIX = 'csys-'
+
+
+class LabelsOperator(str, Enum):
```

Comment:
enum is great but why str?

Reply:
This way we can compare a string with the Enum value. This is a simple test using IPython:
```
In [3]: from enum import Enum

In [4]: class A(Enum):
   ...:     a = 'test'

In [5]: 'test' == A.a
Out[5]: False

In [6]: class A(str, Enum):
   ...:     a = 'test'

In [7]: 'test' == A.a
Out[7]: True
```